docs/how-to/migrate-to-mas.md

```diff
@@ -0,0 +1,145 @@
+# How to back up and restore Synapse
+This document shows how to migrate a legacy synapse homeserver (`latest/edge` track) to an oidc-native homeserver (`2/edge` track).
+
+## Initial deployment setup 
+This document will cover the migration path for a synapse charm on the `latest\edge` track, using the `saml-integrator` charm to provide authentication users via SAML.
+
+## Prepare the new synapse charm
+### Deploy the 2/edge channel on the same model
+We will use the same configuration as the existing synapse charm.
+```
+juju deploy synapse server-mas --channel=2/edge \
+    --config server_name=<server_name> \
+    --config public_baseurl=<public_baseurl>
+```
+
+### Integrate with postgresql
+Integrate with the existing postgresql charm to set up the database for the new charm.
+```
+juju integrate server-mas:mas-database postgresql-k8s
+juju integrate server-mas:database postgresql-k8s
+```
+
+### Configure OIDC
```

Comment:
Some note about external access needed?

Reply:
We added the proxy config to the MAS's pebble layer so I don't think we need to do anything manually anymore